docs/source/project_create.rst

```diff
@@ -137,84 +142,147 @@ After entering your search terms, press enter to start searching.
-modeling quicker.
+.. note::
+
+  It takes relatively long to create a feature matrix with Doc2Vec, but this
+  only has to be done once. The upside of this method is that it takes context
+  into accound. Also, a benifit is the dimension-reduction that generally
+  takes place, which makes the modeling quicker.
+
+Several other feature extractors are available in the software (sentence Bert,
+embedding IDF/LSTM) and more classifiers can be selected via the :doc:`API
+<reference>`, or added via an :ref:`extensions_dev:model extensions`. 
+
+Classifier
+~~~~~~~~~~
+
+The classifier is the machine learning model used to compute the relevance
+scores. The default is Naive Bayes, though relatively simplistic, it seems to
+work quite well on a wide range of datasets. Several other classifiers are
+available in the software (logistic regression, random forest, SVM, LSTM,
+neural net) and more classifiers can be selected via the :doc:`API
+<reference>` or added via an :ref:`extensions_dev:model extensions`. 
+
+The neural nets require `tensorflow <https://www.tensorflow.org/>`_, use
+
+.. code:: bash
+
+    pip install tensorflow
+
+
+.. note::
+
+  Neural nets (and LSTM) require a much larger training set (i.e., more prior knowledge) compared to the other classifiers.
+
+
+
+Balancing Strategy
+~~~~~~~~~~~~~~~~~~
+
+To decrease the class imbalance in the training data, the default is to
+rebalance the training set by a technique called dynamic resampling (DR)
+(`Ferdinands et al., 2020 <https://doi.org/10.31219/osf.io/w6qbg>`_).  DR
+undersamples the number of irrelevant records in the training data, whereas
+the number of relevant records are oversampled such that the size of the
+training data remains the same. The ratio between relevant and irrelevant
+records in the rebalanced training data is not fixed, but dynamically updated
+and depends on the number of records in the available training data, the total
+number of records in the dataset, and the ratio between relevant and
+irrelevant records in the available training data. No balancing or
+undersampling are the other options. Other strategies can be selected via the
+:doc:`API <reference>` or added via an :ref:`extensions_dev:model extensions`.
+
+
+Query Strategy
+~~~~~~~~~~~~~~
+
+The query strategy determines which document is shown after the model has
+computed the relevance scores. The options are: maximum (certainty-based),
+uncertainty, random and clustering. When certainty-based is selected, the
+documents are shown in the order of relevance score. The document most likely
+to be relevant is shown first. When mixed is selected, the next document will
+be selected certainty-based 95% of the time, and uncertainty based or randomly
+chosen otherwise. When random is selected, documents are shown in a random
+order (ignoring the model output completely). Other strategies can be selected
+via the :doc:`API <reference>` or added via an :ref:`extensions_dev:model
+extensions`.
```

Comment:
```suggestion
Query Strategy
~~~~~~~~~~~~~~

The query strategy determines which document is shown after the model has
computed the relevance scores. The options are: maximum (certainty-based),
uncertainty, random, and clustering. When certainty-based is selected, the
documents are shown in the order of relevance score. The document most likely
to be relevant is shown first. When mixed is selected, the next document will
be selected certainty-based 95% of the time, and uncertainty based or randomly
chosen otherwise. When random is selected, documents are shown in a random
order (ignoring the model output completely). Other strategies can be selected
via the :doc:`API <reference>` or added via an :ref:`extensions_dev:model
extensions`.
```